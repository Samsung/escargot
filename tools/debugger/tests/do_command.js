for(let i = 0; i < 3; i++){
	print(i)
}
