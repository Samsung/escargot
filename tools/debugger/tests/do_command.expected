Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_command.js:2
e i
0b do_command.js:2
Breakpoint 1 at tools/debugger/tests/do_command.js:2
c
Print: 0
Stopped at breakpoint:1 tools/debugger/tests/do_command.js:2
c
Print: 1
Stopped at breakpoint:1 tools/debugger/tests/do_command.js:2
(escargot-debugger) c
Print: 2
Connection closed.
